atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/pool/CassandraClientPoolHostLevelMetric.java

```diff
@@ -21,7 +21,8 @@
     NUM_IDLE("numIdle", 0.1, 2.0),
     NUM_ACTIVE("numActive", 0.1, 2.0),
     CREATED("created", 0.01, 2.0),
-    DESTROYED_BY_EVICTOR("destroyedByEvictor", 0.01, 2.0);
+    DESTROYED_BY_EVICTOR("destroyedByEvictor", 0.01, 2.0),
+    EVICTOR_TASK_SIZE("evictorTaskSize", 0.0, 2.0);
```

Comment:
I don't actually know how to configure this properly - the code was really dense. Would welcome some insight into the best way to approach this (and if we can avoid registering this on all deployments).

Reply:
ah right, I'll do that.